### PR TITLE
Update and fix wasm code again, add cli check to .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by expenses**

### CODE REVIEW

File: cli/browser-demo/index.html [hunk 1]
```diff
@@ -16,10 +15,13 @@
 	log('Loading WASM');
 	await init('./pkg/polkadot_cli_bg.wasm');
 	log('Successfully loaded WASM');
+	log('Fetching chain spec');
+	const chain_spec_response = await fetch("https://raw.githubusercontent.com/paritytech/polkadot/master/service/res/westend.json");
```
Comment (tomaka) [inline cli/browser-demo/index.html]: As mentioned in the browser-light-client chat: I don't think that can work, as the format of the database and the network message are directly derived from the runtime's code.

File: cli/src/browser.rs [hunk 2]
```diff
@@ -14,50 +14,40 @@
...
 
-use crate::ChainSpec;
 use log::info;
 use wasm_bindgen::prelude::*;
-use service::IsKusama;
+use browser_utils::{
+	Client,
+	browser_configuration, set_console_error_panic_hook, init_console_log,
+};
+use std::str::FromStr;
```
Comment (tomaka) [inline cli/src/browser.rs]: That import doesn't really seem to be used.

File: cli/src/browser.rs [hunk 3]
```diff
@@ -14,50 +14,40 @@
...
 		.await
 		.map_err(|err| JsValue::from_str(&err.to_string()))
 }
 
-async fn start_inner(chain_spec: String, wasm_ext: browser_utils::Transport) -> Result<browser_utils::Client, Box<dyn std::error::Error>> {
-	browser_utils::set_console_error_panic_hook();
-	browser_utils::init_console_log(log::Level::Info)?;
+async fn start_inner(chain_spec: String, log_level: String) -> Result<Client, Box<dyn std::error::Error>> {
+	set_console_error_panic_hook();
+	init_console_log(log::Level::from_str(&log_level)?)?;
```
Comment (tomaka) [inline cli/src/browser.rs]:
```suggestion
	init_console_log(&log_level.parse()?)?;
```

File: cli/browser-demo/index.html (hunk 1 above)
Comment (tomaka) [reply cli/browser-demo/index.html]: It's also ideologically questionable to download the chainspecs from something that Parity owns, as the entire point of the blockchain client is to be isolated.
If GitHub goes down, the node doesn't work.

Comment (rphmeier) [reply cli/browser-demo/index.html]: The node devs have to be trusted in any case, but I agree that we shouldn't be downloading from GitHub if avoidable. Still, I view in-browser light client as a prototype at the moment, where we can make some allowances as nobody uses it yet :)

**Pull request merged**